com.cubrid.common.ui/src/com/cubrid/common/ui/cubrid/table/dialog/RenameTableDialog.java

```diff
@@ -116,6 +116,8 @@ protected void buttonPressed(int buttonId) {
 				return;
 			}
 			newName = newTableText.getText().trim();
+			//[TOOLS-4464] Table Name are always stored on the server in lower case.
```

Comment:
I think this style of comment is not a CUBRID style, 
 and we can track changes by GitHub, it's better to remove this comment.

Reply:
ok! thanks.